Backend/WebApi.Weather/HourlyUnits.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace WebApi.Weather;
+
+public class HourlyUnits
+{
+    public string? Time { get; set; }
+    public string? Temperature_2m { get; set; }
+
+    public string? Weathercode { get; set; }
+
+    // New Weather models API
+    public string? Weathercode_best_match { get; set; }
```

Comment:
Nazywaj w PascalCase

Reply:
nawiązanie bezpośrednie do nazw w APi

Reply:
To użyj tego https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties?pivots=dotnet-8-0